src/Model/FileParent.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Kirby\Model;
+
+class FileParent
```

Comment:
`Suggestion`: We can use the same placeholder classes in lazy children and files collections. So maybe we could give them a more general name like `FilePlaceholder` or `FileStub`.